client/containers/workflow-pending/helpers/map-pending-task-list.js

```diff
@@ -37,19 +37,22 @@ const mapPendingTaskItem = domain => item => {
       item[PENDING_TASK_TYPE_TO_ID_MAP[pendingTaskType]]
     }`,
     pendingTaskTypeDisplay: PENDING_TASK_TYPE_TO_DISPLAY_MAP[pendingTaskType],
-    ...(runID && { runID: getWorkflowLink({ ...item, domain }) || runID }),
+    ...(runID && {
+      runID: getWorkflowLink({ ...item, clusterName, domain }) || runID,
+    }),
```

Comment:
this hasn't actually changed, but it looks like this is the same as `runID && {runID:...}`?

Reply:
we added clusterName to the list of args